app/views/distributions/new.html.erb

```diff
@@ -50,6 +50,7 @@
                   <%= f.input :issued_at, as: :datetime, ampm: true, minute_step: 15, label: "Distribution date" %>
                   <%= f.input :reminder_email_enabled, as: :boolean, checked_value: true, unchecked_value: false, label: "Send email reminder the day before?" %>
                   <%= f.input :agency_rep, label: "Agency representative" %>
+                  <%= f.input :delivery_method, collection: Distribution.delivery_methods.keys, label_method: :humanize, label: "Delivery method" %>
```

Comment:
Could we change this to radio buttons instead? I think this might be a little easier UX wise to see all options rather than having to click to see them.

Reply:
done